src/agnocast_ioctl_wrapper/src/topic_list.cpp

```diff
@@ -8,42 +8,55 @@
 #include <algorithm>
 #include <iostream>
 #include <string>
+#include <tuple>
 #include <vector>
 
-extern "C" int topic_list()
+namespace
 {
-  // ======== Get Agnocast topics ========
 
+bool is_service_topic(const std::string & topic_name)
+{
+  const std::string prefix = "/AGNOCAST_SRV_";
```

Comment:
Could you define this as global variables in `agnocast_utils.cpp`, and use it here for maintainability and readability?

Reply:
Yes. But i'm starting to think probability of changing the prefix is low, so the current implementation is also fine. I"ll leave it up to you.